vm/mx.vm/mx_vm.py

```diff
@@ -186,3 +187,76 @@ def rm_links(self):
 
     def __str__(self):
         return "Generating GraalVM symlinks in the vm suite"
+
+
+def fetch_labsjdk(args):
+    parser = ArgumentParser(prog='mx fetch-labsjdk')
+    parser.add_argument('--java-version', action='store', help='numeric value representing requested JDK version')
+    parser.add_argument('--java-distribution', action='store', help='ce or ee')
+    parser.add_argument('-q', '--quiet', action='store_true', help='suppress logging output')
```

Comment:
I personally need two more optional flags for this feature to be complete: 1) (only on macOS) `--no-contents-home` that will strip the `Contents/Home` nonsense on macOS, and 2) `--dest-name` as typing `labsjdk-ce-11.0.6+9-jvmci-20.1-b01` in `sdkman` is annoying. I just use `11-ce` as a shorthand.

Reply:
Is the `Contents/Home` prefix that much of a problem if script completely manages the installation of JDKs and setting of `JAVA_HOME`? My concern is that we can run into issues if we make the layout of JDKs not match the default for a platform (and which external tools expect).

Reply:
Not a problem, just a convenience. I use this layout with `sdkman` and all works fine.